picard/track.py

```diff
@@ -220,6 +224,30 @@ def _get_ignored_folksonomy_tags(self):
             tags = [s.strip().lower() for s in ignore_tags.split(',')]
         return tags
 
+    def update_orig_metadata_images(self):
+        class State:
```

Comment:
And also in Album, yes. See https://github.com/metabrainz/picard/pull/657#discussion_r106848223 . If you think State/process_images/update_metadata_images/update_orig_metadata_images should be unified for Track/Album/Cluster in a generic way that works for all use cases, I'll do it, but first confirm that you really want to add such a complex function, please. (The signature for the function would be something similar to ```def update_metadata_images(obj, sources,  update_metadata=True, update_orig_metadata=True)``` and would be used like:
```Metadata.update_metadata_images(self, self.linked_files, False, True)``` in Track